cloudfront/iiif.wellcomecollection.org/terraform/cf_behaviours.tf

```
@@ -0,0 +1,44 @@
+locals {
+  wellcome_images_path_patterns = [
+    "image/V00*",
+    "image/L00*",
+    "image/M00*",
+    "image/B00*",
+    "image/N00*",
+    "image/A00*",
+    "image/W00*",
+    "image/S00*"
+  ]
```

Minor: could we sort these alphabetically? The order looks arbitrary; alphabetical is slightly more readable.

```suggestion
  wellcome_images_path_patterns = [
    "image/A00*",
    "image/B00*",
    "image/L00*",
    "image/M00*",
    "image/N00*",
    "image/S00*"
    "image/V00*",
    "image/W00*",
  ]
```

It also makes it easier to compare these to a list of prefixes from Miro – e.g. AS or FP. Do we need to include those as well?

---

> Do we need to include those as well?
There don't seem to be any images that require registration in the AS/FP prefix range, so they wouldn't strictly be required here.